### ISSUE TITLE
Editorial

### ISSUE THREAD

**Issue by vmmiller** [w3c/wai-bcase#48]
2 paragraphs above "Increase market research", I think a dash is missing between accessibility and focused.

accessibility-focused

Oops... just realised it's a quote so forget the above.
